equinity-historical-data/src/test/java/uk/gov/justice/laa/crime/equinity/historicaldata/controller/Crm14ControllerTest.java

```diff
@@ -53,11 +55,16 @@ class Crm14ControllerTest {
     @Autowired
     Crm14AttachmentService crm14AttachmentService;
 
+    @MockBean
+    Crm14PSEMessagesService crm14PSEMessagesService;
+
```

Comment:
I would had preferred if anything, to add unit test to the service itself.
Mocking the service is not actually testing the service.


Update: This does not need to be a MockedBean. A MockBean needs a when->return clause in order to be mocked. You are mocking data nor service.. so it is fine to just use it as a normal bean

Reply:
Removed Controller tests and added service tests